docs/DeveloperGuide.adoc

```diff
@@ -26,18 +26,11 @@ Refer to the guide <<SettingUp#, here>>.
 === Architecture
 
 .Architecture Diagram
-image::ArchitectureDiagram.png[]
+image::ArchitectureDiagram_FlashCardPro.png[]
```

Comment:
I guess, you forgot to add the image? This is the same with some of other missing diagrams

I have attached a screenshot for your reference:
![image](https://user-images.githubusercontent.com/39835686/67543620-ea1a8280-f724-11e9-8985-b2bf52187b79.png)